libraries/chain/wasm_interface.cpp

```diff
@@ -155,14 +155,18 @@ class privileged_api : public context_aware_api {
          });
       }
 
-      uint32_t get_blockchain_parameters_packed( array_ptr<char> packed_blockchain_parameters, size_t datalen) {
+      uint32_t get_blockchain_parameters_packed( array_ptr<char> packed_blockchain_parameters, size_t buffer_size) {
```

Comment:
Need a doxygen comment in privileged.h to indicate that the behavior for this function is to copy the buffer only if the buffer_size is sufficiently large.

Reply:
Done.